spark-shell/src/main/scala/org/apache/mahout/sparkbindings/shell/MahoutSparkILoop.scala

```diff
@@ -48,55 +77,63 @@ class MahoutSparkILoop extends SparkILoop {
 
     conf.set("spark.executor.memory", "1g")
 
-    sparkContext = mahoutSparkContext(
+    _interp.sparkContext = mahoutSparkContext(
       masterUrl = master,
       appName = "Mahout Spark Shell",
       customJars = jars,
       sparkConf = conf
     )
 
-    echo("Created spark context..")
+    echoToShell("Created spark context..")
     sparkContext
   }
 
+  // need to change our SparkDistributedContext name to 'sc' since  we cannot override the
+  // private sparkCleanUp() method.
+  // this is technically not part of Sparks explicitly defined Developer API though
+  // nothing in the SparkILoopInit.scala file is marked as such.
   override def initializeSpark() {
-    intp.beQuietDuring {
-      command("""
+    _interp.beQuietDuring {
+      _interp.interpret("""
 
-         @transient implicit val sdc: org.apache.mahout.math.drm.DistributedContext =
+         @transient implicit val sc: org.apache.mahout.math.drm.DistributedContext =
```

Comment:
What's the rationale behind this rename? 

I'd rather keep it as is. 

My reasoning is, ok spark shells have 'sc' reserved for spark context, so we can't use the same variable name just to avoid clashes between mahout-enabled scripts and just spark-enabled scripts that potentially may expect spark context to be named "sc" (not likely since it is implicit, but still).